### DISCUSSION TITLE
Trading groups

### DISCUSSION THREAD

**Discussion by joequant** [Polygant/OpenCEX#2] [Ideas]
So what I would like to do is set up an order book with trading groups.  The idea is that a user would put in an order and match it only with selected users.  I am trying to set up some hybrid between a central order book and a localbitcoins style system.

**Comment by Stan-Polygant** [comment]
Hi, thank you for the Idea!

Think it's better for P2P when you exchange with a certain person. We have such a module but it's not in the open-source package.
